clients/cloud/README.md

```diff
@@ -6,7 +6,7 @@ This directory includes examples of Kafka client applications connecting to [Con
 |:---------------------------------:|:-----------------------------------------------:|:-----------------------------:|
 | [![](images/java.png)](java/)     | [![](images/python.png)](python/)               | [![](images/go.png)](go/)     |
 | [![](images/scala.png)](scala/)   | [![](images/confluent-cli.png)](confluent-cli/) | [![](images/ruby.png)](ruby/) |
-| [![](images/groovy.png)](groovy/) | [![](images/kotlin.png)](kotlin/)               |                               |
+| [![](images/groovy.png)](groovy/) | [![](images/kotlin.png)](kotlin/)               | [kafkacat](kafkacat/)         |
```

Comment:
Where is the kafkacat original art you promised me, Yeva?

Reply:
Genius art takes time...